config/vendors.yml

```diff
@@ -0,0 +1,73 @@
+tribal:
+  - B84
+  - C10
+  - K12
+  - K60
+  - K84
+  - L24
+  - L46
+  - L68
+  - N36
+  - O33
+  - S21
+  - S64
+  - S72
+  - 3S1
+  - G50
+  - R12
+  - S18
+  - W20
+  - U80
+  - B32 # also runs ellucian
```

Comment:
Wait, is this a fly in the ointment? will the migration end up marking them with the last provider in the YAML file?

Reply:
Hmm, yes. Tempted to mark them as tribal, if this is the direction they are going.

Reply:
Whichever they use now. Unless they truly have two integrations?